arbnode/node.go

```diff
@@ -281,6 +281,7 @@ func DeployOnL1(ctx context.Context, parentChainReader *headerreader.HeaderReade
 
 type Config struct {
 	Sequencer           bool                        `koanf:"sequencer"`
+	Espresso            bool                        `koanf:"espresso"`
```

Comment:
It doesn't look like any logic (besides the validation below) depends on this flag, so can we remove it?

Reply:
This is used to avoid the breaking of the validation since we need to set `Sequencer` to true and set `Coordinator` to false.